influxdb_iox/tests/end_to_end_cases/management_api.rs

```diff
@@ -447,6 +447,90 @@ async fn test_create_get_update_delete_restore_database() {
     );
 }
 
+#[tokio::test]
+async fn disown_database() {
+    test_helpers::maybe_start_logging();
+    let server_fixture = ServerFixture::create_shared(ServerType::Database).await;
+    let mut client = server_fixture.management_client();
+
+    let db_name = rand_name();
+    let rules = DatabaseRules {
+        name: db_name.clone(),
+        ..Default::default()
+    };
+
+    // Create a database on one server
+    let created_uuid = client.create_database(rules.clone()).await.unwrap();
+
+    // Disown database returns the UUID
```

Comment:
```suggestion
    // Delete database returns the UUID
```
The command below this calls `delete_database`, not `disown_database` -- is that expected?